pkg/cmd/down/wait.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package down
```

Comment:
I created this package to start moving logic outside of the main package, following `kubectl`'s  pattern. I think it's easier if we leave the user-interaction stuff in `main`  (cmd parsing, param matching, input - output) and start moving the logic to packages like this.